modules/applications/pages/ui-builder/faq.adoc

```diff
@@ -141,6 +141,10 @@ Yes, they are compatible with both deployment modes.
 Yes, it is. Reach out to your customer service representative to explore how you can migrate to Bonita Cloud.
 
 
+=== Can Bonita UI Builder be used offline? 
```

Comment:
Did you check this? UIB is downloading resources (images, fonts, ...) from remote servers and therefore cannot be used offline. 
Perhaps you launched UIB once online, then restarted offline ? In this case it may have taken the resources from the cache.

Reply:
Actually, during my tests, I was able to call different API resources and I even was able to instantiate processes. I couldn't see any big difference with the online mode.

Internet is still mandatory though for the installation (and therefore the first launch) so it's kinda hard to tell whether it's because of cache or not.
I also tried in incognito mode and didn't come across any issues either

Reply:
I did some checking: if you empty your cache (Clear site data From dev tools > Storage), and reload your page, you don't get anything. Even without emptying the cache, I noticed side effect (loading delays, ...).

Reply:
I did some investigation and discovered the static files (images, fonts, css, ...) are actually stored at build time in the Appsmith server ! So forget my previous comment then.
But we should at least say that some features such as git cannot be used offline.

Reply:
Awesome, I'll make the changes